### PR TITLE
Stage

### PR DESCRIPTION
**Pull request by rililive**
We should start targeting `stage` when we make PRs. After the stage build is successful, then we make a PR to merge it to master.

This is a work in progress. Still trying to fix one error during the build process

**Update**
All set. Everything seems to be working now. The idea is that we merge feature branches to `stage`, wait for a passing build, then merge them to `master`.

### CODE REVIEW

**Comment by rililive** [comment]
**Working `stage` build**
Screenshot 2023-06-17 at 5 35 50 PM

File: _bin/cicd/deploy.sh [hunk 1]
```diff
@@ -63,8 +63,9 @@ echo "Resetting VERSIONS.txt"
 cat > VERSIONS.txt <<EOF
 EOF
 
+# TODO: Change this to Zach Owen's user
 git config user.email "rili.main@gmail.com"
 git config user.name "Rili Admin"
```
Comment (rililive) [inline _bin/cicd/deploy.sh:68]: Could you update the email and name to your github user?

File: _bin/cicd/publish.sh [hunk 2]
```diff
@@ -53,6 +53,7 @@ cat > VERSIONS.txt <<EOF
 LAST_PUBLISHED_GIT_SHA=${GIT_SHA}
 EOF
 
+  # TODO: Change this to Zach Owen's user
   git config user.email "rili.main@gmail.com"
   git config user.name "Rili Admin"
```
Comment (rililive) [inline _bin/cicd/publish.sh:58]: Could you update the email and name to your github user?

**Review by BlueRacoon — APPROVED** [review]
Looks good!

**Pull request merged**